### ISSUE TITLE
The audio of all videos in the app is very low, close to inaudible. It goes up once the recording activity is started

### ISSUE THREAD

**Issue by rekhanr** [MusicAsLanguage/mobileapp#177]
Launch the app
Start the intro video, the volume is at max
Very low volume
Got to Lesson 2 -> Practice
Start recording and the audio jumps up. Now all videos have the high volume

**Comment by linsungc** [comment]
This looks to be caused by the new change that sets allowsRecordingIOS to support practice mode. The behavior seems to be whenever user visits an activity, coming back would cause audio to go really low. 

Simple repro steps would be, play an intro video, observe the sound is normal; open an activity, do nothing and go back, play the intro video again, now the audio becomes really low. 

Created a PR (#178) to address it by setting & resetting the flag before & after the video recording. Tested the change with few scenarios and looks to be functioning as expected.

Will close the issue once the PR is merged into main.
